omnibus/files/private-chef-cookbooks/private-chef/recipes/nginx.rb

```diff
@@ -125,6 +130,7 @@
  'routes.lua',
  'dispatch_route.lua',
  'check_token_using_endpoint.lua',
+ 'check_token_using_endpoint_required_recipe.lua',
```

Comment:
I wonder if there would be a way to make the existing `check_token_using_endpoint.lua` generic enough to cover this case, too?